src/sqlfluff/dialects/dialect_postgres.py

```diff
@@ -3054,3 +3057,50 @@ class DropPolicyStatementSegment(BaseSegment):
         Ref("TableReferenceSegment"),
         OneOf("CASCADE", "RESTRICT", optional=True),
     )
+
+
+@postgres_dialect.segment()
+class ListenStatementSegment(BaseSegment):
+    """A `LISTEN` statement.
+
+    As Specified in https://www.postgresql.org/docs/current/sql-listen.html
```

Comment:
Rather than use "current" is it better to specify the version implemented here? So if it's found to be missing something we can know why?